litt/src/tracker.rs

```diff
@@ -109,15 +110,43 @@ impl IndexTracker {
         Ok(self.indices.clone())
     }
 
+    pub fn store_fast_results(
+        &self,
+        fast_results: HashMap<u32, (String, u32, String)>,
+    ) -> Result<()> {
+        let base_path = PathBuf::new()
+            .join("~/")
```

Comment:
beware, not compatible with windows

Reply:
I thought the shellexpand would handle the `~` automatically depending on the platform. But maybe I'm mistaken...